src/resources/Microsoft.Authorization/policyDefinitions/Deploy-Private DNS zone-PaaS-Private endpoint-Allowed values.json

```diff
@@ -0,0 +1,233 @@
+    {
+    "name": "Configure a private DNS Zone ID for Paas services-Generic policy for PaaS services dosent has builtin policies",
+    "type": "Microsoft.Authorization/policyDefinitions",
+    "apiVersion": "2021-06-01",
+    "scope": null,
+    "properties": {
+        "policyType": "Custom",
+        "mode": "All",
+        "displayName": "Configure a private DNS Zone ID for Paas services-Generic policy for PaaS services dosent has builtin policies",
+        "description": "Configure private DNS zone group to override the DNS resolution for PaaS services private endpoint",
+        "metadata": {
+        "version": "1.1.0",
+        "category": "Generic",
+        "source": "https://github.com/Azure/Enterprise-Scale/",
+        "alzCloudEnvironments": [
+            "AzureCloud"
+        ]
+        },
+        "parameters": {
+        "effect": {
+            "type": "String",
+            "metadata": {
+            "displayName": "Effect",
+            "description": "Enable or disable the execution of the policy"
+            },
```

Comment:
Please fix indents to properly align with sections. You can use the vscode extension "Prettify JSON" to help with this.